rust/catalyst-voting/src/vote_protocol/mod.rs

```diff
@@ -0,0 +1,93 @@
+//! An implementation of the voting protocol described in this [spec](https://input-output-hk.github.io/catalyst-voices/architecture/08_concepts/voting_transaction/crypto/)
```

Comment:
is the intention to be compatible with legacy JOR i.e create valid jor txs? if so, might be worth informally sanity checking the byte output of the signer in cat-core with this output as the former output was tested and works via the load tester.

Reply:
Yes, you are correct it must be compatible with the jorm.
We discussed it with @stevenj, decided to not add any sort of golden test right now, going to test all this stuff with the integration test